### ISSUE TITLE
thin mode: detect timezone

### ISSUE THREAD

**Issue by BayerSe** [oracle/python-oracledb#144]
The thin and the thick mode drivers behave differently with respect to timezones: thin uses UTC, whereas thick detects the local timezone. This is confusing for people migrating from cx_Oracle to the new thin driver, see e.g. https://github.com/oracle/python-oracledb/issues/140.

It would be great if the thin mode driver would detect the local time zone, too, so that the following script always returns the same value, independent of the `thick` flag.

```python
import oracledb

user ="..."
password = "..."
dsn = "..."

thick = False

if thick:
    oracledb.init_oracle_client()

with oracledb.connect(user=user, password=password, dsn=dsn) as connection:
    with connection.cursor() as cursor:
        tz = cursor.execute("SELECT SESSIONTIMEZONE FROM DUAL").fetchone()[0]

print(f"Timezone: {tz}")
```

**Comment by anthony-tuininga** [comment]
This is on our (long) list of things to do. I'll let @cjbj comment on priority!

**Comment by anthony-tuininga** [comment]
Ok. I had a chance to address this. I have pushed a patch that should correct this issue. If you are able to [build from source](https://python-oracledb.readthedocs.io/en/latest/user_guide/installation.html#installing-from-source-code) you can verify that it corrects your issue as well.

**Comment by anthony-tuininga** [comment]
This has been included in version 1.3.0 which was just released.

**Comment by BayerSe** [comment]
Hi @anthony-tuininga; sorry for replying only now. I've tested it, and it works perfectly. Many thanks!
